src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -147,14 +147,15 @@ <h3 hidden$='[[!errorTitle]]'>[[errorTitle]]</h3>
         this.errorText = '';
       },
       doneTapped: function() {
-        if (!this.$.serverConfig.value) {
+        const userInputConfig = this.$.serverConfig.value;
+        if (!userInputConfig) {
           this.showError('',
               'Please paste the output from the installation process before proceeding.');
           return;
         }
         this.showConnection = true;
         this.fire('ManualServerEntered', {
-          userInputConfig: this.$.serverConfig.value
+          userInputConfig: userInputConfig.replace(/\s+/g, '') // Remove whitespace
```

Comment:
Could you move this logic to the handler for `ManualServerEntered`?
We should have as little logic as possible in the UI components.

Reply:
Done in 1426775, thank you for the suggestion.